### PR TITLE
Add integration test for search context GraphQL methods

### PR DESCRIPTION
**Pull request by camdencheek**
This adds integration tests for the search context create, get, update,
and delete endpoints.

### CODE REVIEW

**Review by rvantonder — APPROVED** [review]
rubber stamping because integration tests

File: internal/gqltestutil/search_context.go [hunk 1]
```diff
@@ -46,6 +52,80 @@ mutation CreateSearchContext($input: SearchContextInput!, $repositories: [Search
...
+}
+`
+	variables := map[string]interface{}{
+		"id":           id,
+		"input":        input,
+		"repositories": repos,
+	}
+	var resp struct {
+		Data struct {
+			UpdateSearchContext GetSearchContextResult `json:"updateSearchContext"`
```
Comment (novoselrok) [inline internal/gqltestutil/search_context.go:118]: If we're using `GetSearchContextResult` struct here, we should also specify all of the necessary fields in GQL request to populate it 🙂, copied from above:
```
id
description
spec
autoDefined
repositories {
  repository {
	  name
  }
  revisions
}
```

**Review by novoselrok — APPROVED** [review]
🎉

File: internal/gqltestutil/search_context.go (hunk 1 above)
Comment (camdencheek) [reply internal/gqltestutil/search_context.go:127]: Done!

**Pull request merged**